docs/products/postgresql/reference/pg-long-running-queries.rst

```diff
@@ -0,0 +1,81 @@
+PostgreSQL® long-running queries
+===============================
+
+Aiven does not terminate any customer queries even if they run indefinitely.  Long-running queries often cause issues by locking resources and preventing database maintenance tasks.
+
+To identify and terminate such long-running queries, you can do it from either:
+
+* `Aiven console <http://console.aiven.io>`_ / web page
+* `PostgreSQL® shell <https://www.postgresql.org/docs/current/app-psql.html>`_ (``psql``)
+
+
+Aiven console
+^^^^^^^^^^^^^
+In the Aiven web console, you can go to the **Current Queries** tab for your service.
+
+.. image:: /images/products/postgresql/pg-long-running-queries.png
+    :alt: PostgreSQL® service overview tab in Aiven's console
+
+
+On this page, you can click **Terminate** to stop any queries directly.
+
+
+PostgreSQL® shell (psql)
+^^^^^^^^^^^^^^^^^^^^^^^^
+You can login to your service by running on the terminal ``psql <service_uri>``.  Once connected, you can call the following function on the ``psql`` shell::
```

Comment:
We can reference this article for the login part https://docs.aiven.io/docs/products/postgresql/howto/connect-psql.html

Reply:
added on the `login to your service` set of words